src/System.Security.Cryptography.X509Certificates/tests/X509StoreTests.cs

```diff
@@ -151,6 +151,19 @@ public static void AddReadOnlyThrows()
         }
 
         [Fact]
+        public static void AddDisposedThrowsCryptographicException()
+        {
+            using (X509Store store = new X509Store(StoreName.My, StoreLocation.CurrentUser))
+            using (X509Certificate2 cert = new X509Certificate2(TestData.MsCertificate))
+            {
+                store.Open(OpenFlags.ReadWrite);
+
+                cert.Dispose();
+                Assert.ThrowsAny<CryptographicException>(() => store.Add(cert));
```

Comment:
This could be Throws instead of ThrowsAny since we expect it to be the specific CryptographicException and not a platform-dependent one with an error code.  But I'm not sure I care for the distinction, so I'm going to offer up that there's a difference, and either change it or leave it is fine with me :smile:.